src/autotrackMemoize/proxy.ts

```diff
@@ -10,11 +10,11 @@ import {
   dirtyTag
 } from './tracking'
 
-export const REDUX_PROXY_LABEL = Symbol()
+export const REDUX_PROXY_LABEL = /* @__PURE__ */ Symbol()
```

Comment:
:question: does the annotation actually do something here?

Reply:
Yes. That symbol somehow stays around even if it's not used. The pure annotation removes it. It's the case with both rollup and webpack.